### PR TITLE
deploy script

### PR DESCRIPTION
**Pull request by drkmen**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -56,6 +56,7 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'rails-controller-testing'
   gem 'faker', git: 'https://github.com/stympy/faker.git', branch: 'master'
+
```
Comment (Tivgres) [inline Gemfile]: Why u need a space at last line before ```end``` ?

File: config/deploy.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# Uncomment the following to require manually verifying the host key before first deploy.
+# set :ssh_options, verify_host_key: :secure
+
+
+namespace :deploy do
+
+  task :restart do
+    on roles :app do
+      within current_path do
+        p '*'*100
```
Comment (Tivgres) [inline config/deploy.rb]: Do u really need it here?

Comment (drkmen) [reply config/deploy.rb]: Yes, i can show you what happens while deploying ;)

File: Gemfile (hunk 1 above)
Comment (drkmen) [reply Gemfile]: indeed

**Pull request merged**